nengo_loihi/tests/test_conv.py

```diff
@@ -460,7 +460,7 @@ def test_conv_connection(channels, channels_last, Simulator, seed, rng, plt, all
         sim_real.run(pres_time)
     real_out = sim_real.data[bp].mean(axis=0).reshape(output_shape.shape)
 
-    with Simulator(model, dt=dt, precompute=False) as sim_loihi:
```

Comment:
Personally, I like being explicit in tests about whether we're testing with `precompute=True` or `precompute=False`. It makes it a lot easier to know what is going on in the test.

Reply:
If there were a comment here explaining why it has to be precomputed for test purposes I could understand that, but my understanding/assumption is that it doesn't work with `precompute=True`, but we were defaulting to `precompute=True`, so that seems like an oversight in the splitter rather than something we did intentionally. If we're trying to test something with/without precomputing I would expect this test to be parametrized and tested with both precomputing and not.

Reply:
We agreed to leave this without specifying `precompute`.